Mathlib/MeasureTheory/Measure/Lebesgue/EqHaar.lean

```diff
@@ -79,6 +79,34 @@ theorem Basis.parallelepiped_basisFun (ι : Type*) [Fintype ι] :
     · exact zero_le_one
 #align basis.parallelepiped_basis_fun Basis.parallelepiped_basisFun
 
+ /-- A parallelepiped can be expressed on the standard basis. -/
+ theorem Basis.parallelepiped_eq_map  {ι E : Type*} [Fintype ι] [NormedAddCommGroup E]
```

Comment:
By the way, why can't that be in `MeasureTheory.Measure.Haar.OfBasis`?

Reply:
This result needs `PositiveCompacts.piIcc01` that is only defined a few lines above in this file.

Reply:
Ah of course, sorry :facepalm: